timeline/views.py

```diff
@@ -40,4 +40,21 @@ def post(self, request):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         except serializers.ValidationError as e:
             error_message = get_validation_error_detail(e)
-            return Response({"error": error_message}, status=status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response({"error": error_message}, status=status.HTTP_400_BAD_REQUEST)
+        
+class TimelineDeleteView(generics.DestroyAPIView):
```

Comment:
There is no permission classes yet, we have to add it to prevent non allowed user to access this endpoint